widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -53,7 +53,7 @@ import com.glia.widgets.view.floatingvisitorvideoview.FloatingVisitorVideoContai
 import com.glia.widgets.view.head.BadgeTextView
 import com.glia.widgets.view.head.controller.ServiceChatHeadController
 import com.glia.widgets.view.header.AppBarView
-import com.glia.widgets.view.unifiedui.exstensions.*
+import com.glia.widgets.view.unifiedui.extensions.*
```

Comment:
Are we using wildcard imports? I have seen this in multiple places and forgot to ask before

Reply:
There is no agreement about it, I especially let the decision to Android studio)